markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -11,414 +11,298 @@ If you create a new module with the goal of contributing the code to *nf-core*,
-
-```nextflow
-input:
-    tuple val(meta), path(reads)
-    val(read_structure)
-
-output:
-    tuple val(meta), path("*_umi_converted.bam"), emit: umibam
-    path "*.version.txt"          , emit: version
-```
-
-### Passing options.args
-
-Any optional parameter should be passed within a groovy map called options, and identified as a string within *args*.
-In our specific case, we are not using any optional argument and therefore we will include the module and initialise this with an empty string in our test workflow.
-
-```nextflow
-include { FGBIO_FASTQTOBAM } from '../../../../modules/fgbio/fastqtobam/main.nf' addParams( options: [args: ''] )
-```
-
-But we will add this into the script code, so any other user who might want to execute the function with additional arguments will be able to.
-The code of the script will therefore look like this, once we have substituted inputs and outputs as appropriate.
-
-```bash
-mkdir tmpFolder
-
-fgbio --tmp-dir=${PWD}/tmpFolder \\
-FastqToBam \\
--i ${reads} \\
--o "${prefix}_umi_converted.bam" \\
---read-structures $read_structure \\
---sample ${meta.id} \\
---library ${meta.id} ${options.args}
-```
-
-Before wrapping up our code, we need to add a line to output the software version.
-
-Usually a software prints their version with a code similar to this
-
-```bash
-tool --version >${software}.version.txt
-```
-
-However, in some cases the software outputs the version as *stderr* and causes an exit that is recognised by Nextflow as if the process ended with an error.
-
-In order to avoid that, we can in general print the version as part of an echo statement, like this
-
-```nextflow
-echo \$(tool --version 2>&1) >${software}.version.txt
-```
-
-Notice the escape `\$` of the first `$` sign to distinguish between *bash* variables and *nextflow* variables.
-
-Unfortunately, FGBIO manages to cause an error exit even with this solution, and we are therefore forced to output the version as a string, like this
+### Test data
 
-```nextflow
-def VERSION = '1.3.0'
-```
+Even before beginning the development of a module, you should identify a small dataset you can use to test its functionality. Ideally, the dataset is existing already and can be shared with other test workflows for other modules.
 
-and then in the script
+> :recycle: This is in active development, keep an eye for available test data [here](https://github.com/nf-core/test-datasets/tree/modules/data) and how to access them using a config file (see this [change](https://github.com/nf-core/modules/pull/365)).
 
-```bash
-echo $VERSION >${software}.version.txt
-```
+---
 
-Our final script will therefore look like this:
+## Writing a new module
 
-```nextflow
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName } from './functions'
+Check out [Tutorial: Create a DSL2 Module](https://nf-co.re/developers/tutorials/dsl2_modules_tutorial) for how to create a new DSL2 Module
```

Comment:
the link points to the old one but I suppose that's the result of the old page being up and the new not rendered yet, so on the gitpod version we open that one.
however, when this page is rendered, wouldn't this link point to itself?